src/Actions/PermissionsManager.php

```diff
@@ -101,6 +114,119 @@ public function setupInstitutionalFilters(): void
         do_action('pb_institutional_filters_created', $institution, $institutionalManagers, $institutionalUsers);
     }
 
+    public function addCustomTextForUsersList(array $customText): array
+    {
+        $institutionId = get_institution_by_manager();
+        if (! is_super_admin() || $institutionId === 0) {
+            return $customText;
+        }
+
+        $institution = Institution::find($institutionId);
+        $totalUsers = InstitutionUser::query()->byInstitution($institutionId)->count();
```

Comment:
I'd replace these 2 queries with a single one like so:

```suggestion
        $institution = Institution::query()
            ->where('id', $institutionId)
            ->withCount('users')
            ->first();
```

That would allow us to do something like `$institution->useres_count` down below